app/assets/javascripts/discourse/app/components/post-stream.gjs

```diff
@@ -249,6 +271,22 @@ export default class PostStream extends Component {
         topicId=@topic.id
       }}
     >
+      {{#if @postStream.canPrependMore}}
```

Comment:
The region will be "visible" even when more posts are being currently loaded. Is this by design or does it make sense to move the region to inside the `if` below the load more component?